aidb/query/query.py

```diff
@@ -148,10 +148,47 @@ def table_and_column_aliases_in_query(self):
           col_name = node.args['this'].args['this'].args['this']
           column_alias_to_name[str.lower(col_alias)] = str.lower(col_name)
 
-
     return table_alias_to_name, column_alias_to_name
 
 
+  def udf_alias_in_query(self):
+    """
+    Find alias of user defined function in query
```

Comment:
Why would a UDF have an alias?

Reply:
https://github.com/ddkang/aidb/blob/2cef721f5c905c44c89d800515d8d7b5536e078c/tests/tests_full_scan_engine_udf.py#L296-L308
The aliases for udf outputs. Users can do filter based on outputs. I will change this function name